### PR TITLE
updated erp2.0 spec to include responsibility channels

### PR DESCRIPTION
**Pull request by iamn1ck**
First draft

### CODE REVIEW

File: erp-data-exchange-specification.md [hunk 1]
```diff
@@ -261,6 +262,29 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
...
+| **importType**   | [Import Type](#import-type)     | One of:<br/>`insert`, or `delete`                                                          |
+| **agentId**      | [UUID](#uuid-type)              | The foreign UUID of the agent that is responsible for the channel path (_agents.csv_ `id`) |
+| **resourceId**   | [UUID](#uuid-type)              | The foreign UUID of the resource that the agent belongs to ([resource].csv)                |
+| **resourceType** | [Resource Type](#resource-type) | The type of the resource being referenced by the resourceId (`property/group/unit`)        |
+
+#### Example
+
+```csv
+importType,agentId,resourceId,resourceType
+insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,ab463b8b-a76c-4f6a-a726-75ab5730b69b, property
```
Comment (peinwag) [inline erp-data-exchange-specification.md:282]: maybe just a stupid question, but once you call the api the uuid (externalId) is converted to our id via the id-lookup right?

File: erp-data-exchange-specification.md [hunk 2]
```diff
@@ -261,6 +262,29 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
...
+The `responsibilityChannels.csv` file describes the relation between an agent and a resource channel
+
+#### Fields
+
+| Field            | Type                            | Description                                                                                |
+| ---------------- | ------------------------------- | ------------------------------------------------------------------------------------------ |
+| **importType**   | [Import Type](#import-type)     | One of:<br/>`insert`, or `delete`                                                          |
+| **agentId**      | [UUID](#uuid-type)              | The foreign UUID of the agent that is responsible for the channel path (_agents.csv_ `id`) |
+| **resourceId**   | [UUID](#uuid-type)              | The foreign UUID of the resource that the agent belongs to ([resource].csv)                |
+| **resourceType** | [Resource Type](#resource-type) | The type of the resource being referenced by the resourceId (`property/group/unit`)        |
```
Comment (peinwag) [inline erp-data-exchange-specification.md]: not sure if we should already expose that we are able to handle other resources except the property. should maybe be clarified with @davgun29  first

**Review by peinwag — CHANGES_REQUESTED** [review]
Left one stupid question + one thing to clarify w. the PO first. Other than that it's fine for me

File: erp-data-exchange-specification.md [hunk 3]
```diff
@@ -261,6 +262,29 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
 insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
 ```
 
+### responsibilityChannels.csv
+
+The `responsibilityChannels.csv` file describes the relation between an agent and a resource channel
+
```
Comment (adashst) [inline erp-data-exchange-specification.md:268]: Maybe we should clarify what a `resource channel` is. This term isn't used here before.
Or just replace it by `resource`.

File: erp-data-exchange-specification.md [hunk 4]
```diff
@@ -261,6 +262,29 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
...
+#### Fields
+
+| Field            | Type                            | Description                                                                                |
+| ---------------- | ------------------------------- | ------------------------------------------------------------------------------------------ |
+| **importType**   | [Import Type](#import-type)     | One of:<br/>`insert`, or `delete`                                                          |
+| **agentId**      | [UUID](#uuid-type)              | The foreign UUID of the agent that is responsible for the channel path (_agents.csv_ `id`) |
+| **resourceId**   | [UUID](#uuid-type)              | The foreign UUID of the resource that the agent belongs to ([resource].csv)                |
```
Comment (adashst) [inline erp-data-exchange-specification.md]:
```suggestion
| **resourceId**   | [UUID](#uuid-type)              | The foreign UUID of the resource that the agent belongs to ([resource](#resource-type).csv)                |
```

File: erp-data-exchange-specification.md (hunk 1 above)
Comment (iamn1ck) [reply erp-data-exchange-specification.md:282]: ya

File: erp-data-exchange-specification.md [hunk 5]
```diff
@@ -261,6 +262,34 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
...
+
+The `responsibilityChannels.csv` file describes the relation between an agent and a resource channel
+
+#### Fields
+
+| Field            | Type                            | Description                                                                                |
+| ---------------- | ------------------------------- | ------------------------------------------------------------------------------------------ |
+| **importType**   | [Import Type](#import-type)     | One of:<br/>`insert`, or `delete`                                                          |
+| **agentId**      | [UUID](#uuid-type)              | The foreign UUID of the agent that is responsible for the channel path (_agents.csv_ `id`) |
+<<<<<<< Updated upstream
```
Comment (peinwag) [inline erp-data-exchange-specification.md]: leftovers from merging / stashing

**Review by peinwag — APPROVED** [review]
just left a comment about the leftovers from merging/stashing other than that LGTM

File: erp-data-exchange-specification.md (hunk 3 above)
Comment (adashst) [reply erp-data-exchange-specification.md:268]: Other that this LGTM

**Review by adashst — APPROVED** [review]
Other than thew mentioned thing about resource-channel -> resource.
LGTM

**Comment by adashst** [comment]
@iamn1ck Thanks a lot. Can we merge this now?

**Pull request merged**